app/components/forms/admin/settings/billing.js

```diff
@@ -0,0 +1,124 @@
+import Component from '@ember/component';
+import FormMixin from 'open-event-frontend/mixins/form';
+
+export default Component.extend(FormMixin, {
```

Comment:
Lets start moving to ES6 style classes for all new resources. 
https://github.com/fossasia/open-event-frontend/pull/3169/files